saos-persistence/src/main/java/pl/edu/icm/saos/persistence/search/dto/LawJournalEntrySearchFilter.java

```diff
@@ -0,0 +1,101 @@
+package pl.edu.icm.saos.persistence.search.dto;
+
+import pl.edu.icm.saos.persistence.model.LawJournalEntry;
+
+/**
+ * @author madryk
+ */
+public class LawJournalEntrySearchFilter extends DatabaseSearchFilter<LawJournalEntry> {
+
+    private static final long serialVersionUID = 1L;
+    
+
+    public static class Builder extends DatabaseSearchFilter.Builder<Builder, LawJournalEntrySearchFilter>{
```

Comment:
I would move this Builder to the end of the class under the INNER CLASSES section.